src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 0 16px;
+  border-radius: 5px;
+  max-width: 198.4px;
```

Comment:
I updated code yesterday. All tests work, but i`m not sure that line 9 is correct, without it width of card is 201.6px instead of 200px

Reply:
fixed everything